### ISSUE TITLE
Update The Menu

### ISSUE THREAD

**Issue by Wizardorzo** [CEOSimon/Wrist-Menu-modified#9]
The changing other people's colors won't work. Please fix it.

**Comment by OrkaGT** [comment]
it's patched. stop trying to download these kind of mods

**Comment by CEOSimon** [comment]
Thank god someone said it I keep getting replies to a mod that came out
months ago

**Comment by MellowFishy** [comment]
are the mods still patched?

**Comment by CEOSimon** [comment]
Yes and they won’t be unpatched game is utter trash and I won’t be
returning to mod it
